config.json

```diff
@@ -2806,6 +2806,34 @@
           "list-comprehensions"
         ],
         "difficulty": 7
+       },
+       {
+        "slug": "react",
+        "name": "React",
+        "uuid": "e4462e0c-b396-459c-815a-993a0f04c46a",
+        "prerequisites": [
+          "atoms",
+          "tuples",
+          "lists",
+          "enum",
+          "case",
+          "cond",
+          "if",
+          "multiple-clause-functions",
+          "pattern-matching",
+          "anonymous-functions",
+          "default-arguments",
+          "guards",
+          "maps",
+          "structs",
+          "pids",
+          "processes",
+          "recursion"
+        ],
+        "practices": [
+          "processes"
```

Comment:
I made a note in https://github.com/exercism/elixir/issues/559 that this will be a genserver exercise once we have that concept 🙂